releasenotes/notes/remove-variational-form-3ff4901bdde9ab1e.yaml

```diff
@@ -0,0 +1,5 @@
+---
+deprecations:
+  - |
+    The legacy class `VariationalForm` and option `preferred_init_point` were
+    removed.
```

Comment:
A release not is not needed here - the logic coming from Aqua, however it got refactored before this first release out of Terra, will be addressed in a migration guide